test/relations.test.js

```diff
@@ -835,6 +835,23 @@ describe('relations', function() {
         });
       });
 
+      context('findById with filter include that contains scope with string filter fields', function() {
```

Comment:
change this to `context('findById with include filter that contains string fields')`